### PR TITLE
Task/eparker71/tlt 2321 add lti permissions to account admin

### PR DESCRIPTION
**Pull request by eparker71**
added the lti_permission_required decorator to views
updated version of icommons_commons from 1.10.12 to 1.10.19
added lti_permissions to project
added permission key to base.py
removed old permission checks in dashboard template
added sql file with init permissions (only Account Admins and School Liaisons have access)

### CODE REVIEW

File: canvas_account_admin_tools/requirements/base.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ django-cached-authentication-middleware==0.2.1
 django-proxy==1.0.2
 django-redis-cache==1.5.3
 git+ssh://git@github.com/Harvard-University-iCommons/django-auth-lti.git@v1.2.4#egg=django-auth-lti==1.2.4
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.10.12#egg=django-icommons-common==1.10.12
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.10.19#egg=django-icommons-common==1.10.12
```
Comment (sapnamysore) [inline canvas_account_admin_tools/requirements/base.txt]: can u update this to read 1.10.19 for the #egg too?

**Comment by sapnamysore** [comment]
@smysore

File: course_info/views.py [hunk 2]
```diff
@@ -15,7 +14,7 @@
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
```
Comment (sapnamysore) [inline course_info/views.py]: Per the pow-wow we had, lets leave the role check in there in addition to adding this permission check

File: course_info/views.py [hunk 3]
```diff
@@ -29,7 +28,7 @@ def index(request):
...
```
Comment (sapnamysore) [inline course_info/views.py]: same as above

File: course_info/data_migrations/canvas_account_admin_perms.sql [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	('account_admin_tools', '*', 'Help Desk', FALSE),
+	('account_admin_tools', '*', 'Librarian', FALSE),
+	('account_admin_tools', '*', 'ObserverEnrollment', FALSE),
+	('account_admin_tools', '*', 'SchoolLiaison', TRUE),
+	('account_admin_tools', '*', 'Shopper', FALSE),
+	('account_admin_tools', '*', 'StudentEnrollment', FALSE),
+	('account_admin_tools', '*', 'TaEnrollment', FALSE),
+	('account_admin_tools', '*', 'TeacherEnrollment', FALSE),
+	('account_admin_tools', '*', 'Teaching Staff', FALSE);
+
```
Comment (sapnamysore) [inline course_info/data_migrations/canvas_account_admin_perms.sql]: sql is good. We could check if devops is good with .sql or needs a migrations file for this.

File: canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html [hunk 5]
```diff
@@ -56,24 +51,13 @@ <h4 class="list-group-item-heading">Courses in this Account</h4>
...
-    {% if not has_manage_courses and not has_manage_account %}
-    <div class="col-md-6">
-        <div class="panel panel-default">
-            <div class="panel-body">
-                <p>There are no administration tools installed in this account.</p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+
```
Comment (sapnamysore) [inline canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html]: Will user het a blank screen now if there are no permissions?

Comment (sapnamysore) [inline canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html]: Should we open tech-debt to replace this message?

**Comment by sapnamysore** [comment]
All set :+1:

File: canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html (hunk 5 above)
Comment (eparker71) [inline canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html]: They should get permission denied message, we can test and add ticket if that's not desired behavior.

File: course_info/views.py (hunk 2 above)
Comment (eparker71) [inline course_info/views.py]: added @lti_role_required(const.ADMINISTRATOR) check back per pow wow :)

File: course_info/views.py (hunk 3 above)
Comment (eparker71) [inline course_info/views.py]: added back

**Pull request merged**